CHANGELOG.md

```diff
@@ -4,23 +4,42 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
-## Latest
+## Unreleased
```

Comment:
Is that a common practice to us mention features not included in the release?

Reply:
Since this PR is to master branch, it is not good idea to remove in this PR. I believe it will be removed in the cherry pick of this commit to 2.0.x branch.

Reply:
yes the cherry-pick will not include the parts about unreleased versions